freanalysis_radiation/freanalysis_radiation/__init__.py

```diff
@@ -26,6 +26,7 @@ class Metadata:
     chunk_freq: str = ""
     platform: str = ""
     cell_methods: str = ""
+    chunk_freq: str = "P1Y"
```

Comment:
This will also change in the future, possibly/probably.

Is this really needed though? Does the analysis actually require 1-year timeseries chunks?